activiti-api-process-model/src/main/java/org/activiti/api/process/model/builders/GetSubprocessesPayloadBuilder.java

```diff
@@ -0,0 +1,23 @@
+package org.activiti.api.process.model.builders;
+
+import org.activiti.api.process.model.ProcessInstance;
+import org.activiti.api.process.model.payloads.GetSubprocessesPayload;
+
+public class GetSubprocessesPayloadBuilder {
```

Comment:
I don't think you need to create a new payload type for subprocess. You can reuse `GetProcessInstancesPayload` and add a `parentProcessInstanceId` field in there. Then you can use `GetProcessInstancesPayloadBuilder.parentProcessIntanceId(parentProcessInstanceId)`